pkg/imgpkg/image/tar_image.go

```diff
@@ -180,3 +181,119 @@ func (i *TarImage) isExcluded(relPath string) bool {
 	}
 	return false
 }
+
+// CreateOciTarFileAndDeleteFolder creates a oci tar file from the source folder and deletes the source folder. Used while pushing the oci-tar
```

Comment:
Why are we deleting the source folder?
If I do `imgpkg push -b repo/app1-config -f config/ --to-oci-tar /path/to/file.tar` is my `config` folder going to be deleted? We should not do that